project_multifamily_beta/housing_characteristics/Cooling Setpoint Has Offset.tsv

```diff
@@ -11,7 +11,6 @@ AZ_Prescott-Love.Field.723723	0.404233	0.595767
 AZ_Tucson.Intl.AP.722740	0.404233	0.595767
 CA_Arcata.AP.725945	0.404233	0.595767
 CA_Bakersfield-Meadows.Field.723840	0.450450	0.549550
-CA_Barstow.Daggett.AP.723815	0.450450	0.549550
```

Comment:
Does this mean we only have 215 locations now? Is this compatible with use of AMYs?

Reply:
You are correct. We have 215 locations. When redefining the locations based on counties, the population centroid and the climate zone number was used in determining a distance function to the closest location. The Long Beach location was a lot closer in this distance function than Barstow Daggett AP because of the population centroid for San Bernardino County is a lot closer to the Los Angeles metropolitan area than Barstow, CA.